FantasyUniverse/Views/QuestionView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  QuestionView.swift
+//  FantasyUniverse
+//
+//  Created by TarasPeregrinus on 01.02.2022.
+//
+
+import SwiftUI
+
+struct QuestionView: View {
+    
+    @Binding var questions: [Question]
+    @State var currentQuestion: Question
+    @Binding var correct: Int
+    @Binding var wrong: Int
+    @Binding var answered: Int
+    @State var selected = ""
+    
+    var body: some View {
+        
+        VStack(spacing: 20) {
+            // Question
+            QuestionLabelView(text: currentQuestion.question!)
```

Comment:
Maybe adding a `QuestionViewData` would be helpful if you have to have optional values in the Firebase model (`Question`). It would allow you to spare yourself the unwraps (we don't like unwraps, right?) ;P If a question is missing a required value you just don't create the view data for it, either ignoring it silently or showing an error.